io-classes/src/Control/Concurrent/Class/MonadMVar.hs

```diff
@@ -3,7 +3,7 @@
 {-# LANGUAGE TypeFamilyDependencies #-}
 {-# LANGUAGE TypeOperators          #-}
 
-module Control.Monad.Class.MonadMVar (MonadMVar (..)) where
+module Control.Concurrent.Class.MonadMVar (MonadMVar (..)) where
```

Comment:
Could you add the `Control.Monad.Class.MonadMVar` and reexport from it the new one but add a deprecation warning to it.